check.py

```diff
@@ -175,6 +175,31 @@ def do_asm2wasm_test():
               fail_with_error('wasm interpreter error: ' + err) # failed to pretty-print
             fail_with_error('wasm interpreter error')
 
+        # verify debug info
+        if 'debugInfo' in asm:
+          jsmap = 'a.wasm.map'
+          cmd += ['--binarymap-file', jsmap,
```

Comment:
> what i am asking is, should we change the parameter names to --source-map or something similar

I can change that, yes. For JSON file name it will be `--source-map`, and for url (embedded in the wasm) `--source-map-url`. Is it correct?